src/main/php/PDepend/Report/Overview/Pyramid.php

```diff
@@ -198,7 +198,12 @@ public function close()
         $proportions = $this->computeProportions($metrics);
 
         $svg = new DOMDocument('1.0', 'UTF-8');
-        $svg->loadXML(file_get_contents(dirname(__FILE__) . '/pyramid.svg'));
+        $templatePath = dirname(__FILE__) . '/pyramid.svg';
```

Comment:
```suggestion
        $templatePath = __DIR__ . '/pyramid.svg';
```
Twice the same modification, this sounds like an opportunity to dedupe the SVG loading. A `private loadSvgFile()` would be nice.

Reply:
They are not in the same file so private at the very least is not an option. Any chance we can take this opportunity another time?